tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -208,17 +192,16 @@ export class WebGPUBackend extends KernelBackend {
    */
   disposeData(dataId: DataId, force = false): boolean {
```

Comment:
Can you also modify this function as [webgl did](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/backend_webgl.ts#L618-L657), specially for the process of `force`? It seems that the currently implementation has problem.

Reply:
done